js/controller.js

```diff
@@ -269,7 +269,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         element = elementVideo;
```

Comment:
Do you need to do any check if "element" exists? A lot of the function depends on it. Also, do you need to do something else if element or element.get(0) doesn't exist or just let it continue on?